### ISSUE TITLE
Review 2019-04 i2cner report

### ISSUE THREAD

**Issue by achubbz** [arfc/i2cner#81]
[2019-04-i2cner-report.pdf](https://github.com/arfc/i2cner/files/3114184/2019-04-i2cner-report.pdf)
(source:https://github.com/arfc/i2cner/tree/master/doc/2019-04-report)
Thank you @katyhuff for reviewing this.

**Comment by achubbz** [comment]
Feedback from @katyhuff : 
- [ ] Fix logo on cover page.
- [ ] Fix plot axes to make sure years are not float.
- [ ] Fix plot axes to make sure they start at the origin.
- [ ] Merge 2018-09 and 2019-04 report.
- [ ] Make sure acronyms are well-defined, use the  ``acros``  package.
- [ ] Cite documents instead of including hyperlinks to them.
- [ ] Include more plots from 2018-10 - 2019 -04.
- [ ] Ensure model readme files are .md
- [ ] Ensure model readme files are up to date.

**Comment by achubbz** [comment]
Also add:
- [ ] Data tables

**Comment by achubbz** [comment]
Report updated:
[2019-05-i2cner-report.pdf](https://github.com/arfc/i2cner/files/3175553/2019-05-i2cner-report.pdf)
Reassigned @katyhuff 

Separate issue created for readmes.

**Comment by katyhuff** [comment]
At first glance:
- [ ] Please reduce the quantity of text couched in bulleted lists. Nearly every section is made entirely of bulleted lists. A report should read like a narrative. So, rather than putting all paragraphs in enumerated or bulleted lists, please just allow them to be paragraphs. See the \paragraph{} command for the formatting you might like.
- [ ] please add [hidelinks] to the \usepackage{hyperref} command. 
- [ ] When citing more than one reference, don't use multiple `\cite{}` commands. Use one cite command, and separate the keys with commas `\cite{lastname_title_YYYY,doe_stuff_2019,chaube_i2cner_2019}` . This will result in [1,2,3] rather than [1][2][3]
- [ ] I usually see LCOE written in all caps, not with the little "o" for the of (LCoE). Same with DOE, DOT, MOAB... 
- [ ] You say ". LCoE for fossil fuels has been held constant throughout the simulation " and you explain how wind and solar is handled wrt LCOE. I assume nuclear LCOE is also held constant?
- [ ] In section 3.1, please describe the fundamentals first "100 years of 
- [ ] Anywhere you say "capacity" make sure you indicate whether this is generation capacity (CF*C) or installed capacity (C). 
- [ ] This is a little rude and not a totally complete story... . "The project was significantly delayed due to the quality of the documenta- tion and customer support provided by the VEDA developers. " Perhaps instead, say: "The project has been delayed in part due to limited documentation and customer support available to VEDA users."

**Comment by katyhuff** [comment]
Incorporating more i2cner technology and Cost Analysis are included as an afterthought in "desired goals." However, these two items are the whole point of the project. I'd say it's somewhat more important than the following "critical goals" which I think are very much second order impacts and will take much more time than they are worth. 

- Replace LCoE in all models with a detailed cost structure
- Incorporation of Japan-specific costs for wind
- Revise CCS costs
- Revise nuclear costs

Please change "Critical Goals" to "Next Steps" and change "Desired Goals" to "Potential Improvements"

**Comment by katyhuff** [comment]
Same comment as above wrt Sensitivity analysis. This is a primary goal, not a tertiary goal.

**Comment by achubbz** [comment]
[2019-05-i2cner-report.pdf](https://github.com/arfc/i2cner/files/3178927/2019-05-i2cner-report.pdf)

**Comment by katyhuff** [comment]
@achubbz did you see my comment about hidelinks?

**Comment by katyhuff** [comment]
Ah, it's not `\usepackage{hyperref}[hidelinks]` 
It's`\usepackage[hidelinks]{hyperref}`

**Comment by achubbz** [comment]
Fixed it. Sorry.
[2019-05-i2cner-report.pdf](https://github.com/arfc/i2cner/files/3178982/2019-05-i2cner-report.pdf)

**Comment by katyhuff** [comment]
- [ ] In Section 3.1, please include a few sentences to introduce and explain the tables, rather than just two untitled tables.  
- [ ] TIMES is a VEDA generator. The report says the opposite. (https://iea-etsap.org/index.php/etsap-tools)
- [ ] Does this sentence "We found the format of the developer-prescribed model files restrictive and unsuitable for our purposes. Therefore, we took the time to develop our own model files, which we have progressively refined since then." Mean that you didn't use TIMES?
- No section should begin with a bare bullet. All bulleted / enumerated lists must be introduced with an explanation/introduction. 
- The tables have no titles/captions. Please add titles/captions for all tables. That is, each table should look like so:
![image](https://user-images.githubusercontent.com/393899/57719055-79831c00-7644-11e9-8b26-57b36a86f741.png)


A handful of the following things [from the writing checklist](http://arfc.github.io/manual/guides/writing/checklist/) should be handled (some more important than others).

- [ ] All figures and tables should be referenced (and discussed!) in the text. 
- [ ] Page 4. Do not use the word "where" unless referring to a location (try "such that" or "in which").
- [ ] get rid of extraneous adverbs (very, really, quite, basically, generally)
- [ ] turn negatives to positives (not taken into account -> neglected)
- [ ] bottom of page 11: No need for this to be a bulleted list. Please consider making this a paragraph. Also, "At the very least" indicates there is some vast terrain of other options that we're ignoring. That's really not the case. So, please delete "At the very least," 
- [ ] avoid turning verbs into nouns ("arrived at estimates of" -> "estimated"; "provides a description of" -> "describes") 
- [ ] don't bury the verb (keep the predicate close to the subject at the beginning of the sentence)
- [ ] data is plural (the data are critical) [ see page 1 "data seems/seem", Page 12 The data for retrofitting in Japan is/are not easily available.]

**Comment by achubbz** [comment]
[2019-05-i2cner-report.pdf](https://github.com/arfc/i2cner/files/3180258/2019-05-i2cner-report.pdf)

From https://iea-etsap.org/index.php/etsap-tools/data-handling-shells/veda :
"Data and assumptions are fed into VEDA_FE that provides input to the TIMES code"
Also, note the diagram: VEDA-FE (input data)----> TIMES (equations) ----> GAMS (solution of equations)

It is not possible to use VEDA and not use TIMES.

Changed "We found the format of the developer-prescribed model files restrictive and unsuitable for our purposes. Therefore, we took the time to develop our own model files, which we have progressively refined since then." to "Using the simplest VEDA input files from the VEDA tutorial [2] as a starting
point, we developed our own model files, which we have been progressively
refining since then"

I appreciate the note about "not burying the verb". I want to better my writing skills. I made a few changes to that effect. But I can't guarantee that I've sanitized the document of all such occurrences because it's a long standing habit and sometimes I don't even realize this is a problem. So if you see such an occurrence, I apologize.
